src/Couchbase.Lite.Tests.Shared/DatabaseTest.cs

```diff
@@ -1259,6 +1259,18 @@ public void TestGetDocFromDeletedDB()
             }
         }
 
+        [Fact]
+        public void TestCloseWithActiveLiveQueries()
+        {
+            WithActiveLiveQueries(true);
+        }
```

Comment:
Active replicator tests?

Reply:
They are in the other PR (Replicator dev) which has all the replication updates.

Reply:
Alright, so CBL-774 can be closed after both of those are merged

Reply:
if 774 is approved, 774 can be merged into master, and I will rebase master into replicator dev branch, and replicator dev can point to master so replicator dev pr will show only replicator updates. cleaner this way..

Reply:
I didn't go look at the other PR. Why does making the replication change in a separate PR? This makes this PR incomplete and is hard to know if anything missing.

Reply:
replication dev PR is a big one, but if you like, we can merge this pr into the replication dev pr. but replication pr will not only contain database close/delete, it will also contain lots of replication updates..

Reply:
Then don't merge.